pointcloud/basic_octree_test.go

```diff
@@ -697,3 +699,99 @@ func TestCachedMaxProbability(t *testing.T) {
 		test.That(t, mp, test.ShouldEqual, -2)
 	})
 }
+
+func TestBasicOctreeTransform(t *testing.T) {
+	center := r3.Vector{X: 0, Y: 0, Z: 0}
+	side := 2.0
+
+	octree, err := createNewOctree(center, side)
+	test.That(t, err, test.ShouldBeNil)
+	pointsAndData := []PointAndData{
+		{P: r3.Vector{X: 0, Y: 0, Z: 0}, D: NewValueData(2)},
+		{P: r3.Vector{X: 1, Y: 0, Z: 0}, D: NewValueData(3)},
+		{P: r3.Vector{X: 1, Y: 1, Z: 1}, D: NewValueData(5)},
+	}
+	err = addPoints(octree, pointsAndData)
+	test.That(t, err, test.ShouldBeNil)
+
+	epsilon := 1e-6 // Account for floating point drift with orientation calculations
```

Comment:
[opt] can we make this a global var for testing. i'd prefer to have it called out at the top of the test file

Reply:
Will do